api/internal/core/store/validate_test.go

```diff
@@ -249,6 +248,30 @@ func TestAPISIXJsonSchemaValidator_checkUpstream(t *testing.T) {
 	assert.EqualError(t, err, "schema validate failed: (root): Does not match pattern '^((uri|server_name|server_addr|request_uri|remote_port|remote_addr|query_string|host|hostname)|arg_[0-9a-zA-z_-]+)$'")
 }
 
+func TestAPISIXJsonSchemaValidator_Plugin(t *testing.T) {
+	validator, err := NewAPISIXJsonSchemaValidator("main.route")
+	assert.Nil(t, err)
+
+	// validate plugin's schema which has no `properties` or empty `properties`
```

Comment:
need E2E test cases @nic-chen

Reply:
@membphis  added.

Reply:
Hmmm, I think we can not put all test case to e2e test, e2e test is too heavy to maintain.So e2e usually used for ensuring key path of the software, If some cases could covered by unit test, just maintain unit test cases is fine.

Reply:
@ShiningRush  we need to ensure the DP works well with `disable` property too.